src/activities/mirrorgame/Mirrorgame.qml

```diff
@@ -0,0 +1,501 @@
+           anchors.fill: parent
+           color:"transparent"
+
+        Canvas{
+            id:solution
+            width:parent.width
+            height:parent.height
+            opacity: 0
+            onPaint:{
+               var context = getContext("2d")
+                context.beginPath();
+                context.lineWidth = 2;
+                context.strokeStyle = "yellow"
+                context.moveTo(torch.x+30,torch.y+60);
+                context.lineTo(mirror2.x+50,mirror2.y+50);
+                context.stroke();
+                context.moveTo(mirror2.x+50,mirror2.y+50);
+                context.lineTo(mirror3.x+50,mirror3.y+50);
+                context.stroke();
+                context.moveTo(mirror3.x+50,mirror3.y+50);
+                context.lineTo(mirror4.x+50,mirror4.y+50);
+                context.stroke();
+                context.moveTo(mirror4.x+50,mirror4.y+50);
+                context.lineTo(mirror1.x+50,mirror1.y+50);
+                context.stroke();
+                context.moveTo(mirror1.x+50,mirror1.y+50);
+                context.lineTo(torch.x+20,torch.y-70);
+                context.stroke();}
+
+
+            }
+
+
+        Image {
+            id: bulboff
+            source:"bulbOFF.svg"
+            fillMode: Image.PreserveAspectFit
+            x:torch.x-20
+            y:torch.y-140
+            width:75
+            height:75
+          }
+        Image {
+            id: bulbon
+            source:"bulbON.svg"
+            visible: false
+            fillMode: Image.PreserveAspectFit
+            x:torch.x-30
+            y:torch.y-155
+            width:100
+            height:100
+
+          }
+
+        Image {
+            id: ok
+            opacity: 0
+            source:"qrc:/gcompris/src/core/resource/bar_ok.svg"
+            fillMode: Image.PreserveAspectFit
+            anchors.horizontalCenter: parent.horizontalCenter
+            PropertyAnimation {
+                id: showOk
+                target: ok
+                properties: "opacity"
+                from: 0
+                to: 1
+                duration: 600
+                }
+            MouseArea {
+                anchors.fill: parent
+                onClicked: {
+                    Activity.win();
+                    lvl1.visible = false;
+                    lvl2.source = "Level2.qml";
+
+
+
+                }
+            }
+        }
+
+        Canvas{
+        id:wire
+        width:parent.width
+        height:parent.height
+        onPaint:{
+            var context = getContext("2d")
+             context.beginPath();
+             context.lineWidth = 4;
+             context.strokeStyle = "black"
+             context.moveTo(torch.x+400,torch.y+10);
+             context.lineTo(torch.x+80,torch.y+10);
+             context.stroke();
+
+        }
+        Image {
+            id: off
+            source:"off.svg"
+            fillMode: Image.PreserveAspectFit
+            x:1180

```

Comment:
You can't use absolute positions, it does not work when resizing the screen (or simply with screen with lower resolution like phones/tablets)